src/main/java/de/muenchen/isi/domain/service/AbfrageListService.java

```diff
@@ -42,4 +43,28 @@ public AbfrageListElementsModel getAbfrageListElements() {
         abfrageListElementsModel.setListElements(listElements);
         return abfrageListElementsModel;
     }
+
+    /**
+     * Die Methode gibt {@link AbfrageListElementModel}e sortiert nach createdDateTime aufsteigend zurück.
+     *
+     * @param bauvorhabenId um Bauvorhaben zu ermitteln
+     * @return die {@link AbfrageListElementsModel}s sortiert in aufsteigender Reihenfolge.
+     */
+    public AbfrageListElementsModel getAbfrageListElementsThatReferenceBauvorhaben(final UUID bauvorhabenId) {
+        final var abfrageListElementsModel = new AbfrageListElementsModel();
+        final List<AbfrageListElementModel> listElements = new ArrayList<>();
+
+        // Holen und Mappen
+        this.abfrageService.getAllReferencedAbfragenForBauvorhaben(bauvorhabenId)
+            .stream()
+            .map(this.abfrageDomainMapper::model2ListElementModel)
+            .forEach(listElements::add);
+
+        // Sortieren
+        listElements.sort(Comparator.comparing(AbfrageListElementModel::getFristStellungnahme));
```

Comment:
Wird die Sortierung bereits in Zeile 58 sichergestellt? Evtl den Methodennamen um die Sortierung erweitern.

Alternativ kann hier gleich das Repo verwendet werden.